CHANGELOG.md

```diff
@@ -8,6 +8,21 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [1.3.1] - 2023-08-02
+
+### Added
+* Count datastore queries involved in Check resolution ([#880](https://github.com/openfga/openfga/pull/880))
+
+  OpenFGA request logs will now include a field `datastore_query_count` that shows how many queries were involved in a single Check resolution.
+
+### Fixed
+* Avoid panic due to incorrect error handling in Check ([#923](https://github.com/openfga/openfga/pull/923))
```

Comment:
We don't need to mention this. This bug was introduced in https://github.com/openfga/openfga/pull/880 which we are including in this release

Reply:
True

Reply:
Dropped.